### PR TITLE
sample contains slide deck not diagram

### PR DESCRIPTION
**Pull request by ecki**
I also added the location of the input source, as it is a non-standard directory and name. I also removed the 5 in HTML5 as I think that is not the main point of the output format.

### CODE REVIEW

**Comment by abelsromero** [comment]
Thanks a lot for noticing the mistakes :+1: 
I just added two comments.

File: asciidoc-to-revealjs-example/README.adoc [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
 
 == Usage
 
-Convert the AsciiDoc to HTML5 by invoking the `process-resources` goal (configured as the default goal):
+Convert the AsciiDoc to HTML slides by invoking the `process-resources` goal (configured as the default goal):
 
  $ mvn
 
-Open the file _target/generated-slides/example-manual.html_ in your browser to see the generated HTML file containing the generated diagram images.
+Open the file _target/generated-slides/slides.html_ in your browser to see the generated HTML file containing the _slides.js_ presentation deck.
```
Comment (mojavelinux) [inline asciidoc-to-revealjs-example/README.adoc]: Isn't it a reveal.js presentation deck?

**Comment by ecki** [comment]
Included your suggestions.

**Comment by mojavelinux** [comment]
:+1:

**Pull request merged**